modules/social_features/social_content_report/social_content_report.module

```diff
@@ -119,16 +119,10 @@ function social_content_report_comment_links_alter(array &$links, CommentInterfa
  */
 function social_content_report_node_links_alter(array &$links, NodeInterface $entity, array &$context) {
   if ($context['view_mode'] === 'full') {
-    $link = \Drupal::service('social_content_report.content_report_service')->getModalLink($entity, 'report_node', TRUE);
+    $link_report = \Drupal::service('social_content_report.content_report_service')->getModalLink($entity, 'report_node', TRUE);
 
-    if ($link) {
-      $links['moderation'] = [
-        '#theme' => 'links__node__moderation',
-        '#links' => [
-          'report' => $link,
-        ],
-        '#attributes' => ['class' => ['links', 'inline']],
-      ];
+    if ($link_report) {
```

Comment:
@VBaldych I am not sure if such refactoring was really needed, but if so I think we don't need to create new helper service in `social_core`, but better to use existing one `social_content_report.content_report_service`

Reply:
@ribel my service is generic one. I decided to add it in **social_core** because:
1. We use this service in both modules **social_album** and **social_content_report**
2. Both of this modules already have dependency on **social_core**

Reply:
Ok, but in **Album** module we want to render **Report** link, so in that case we can also add dependency from `social_content_report` to the `social_album`.

P.S. @ronaldtebrake, @Kingdutch  What do you think, guys?

Reply:
No need. **Report** link being adding to independently. If 'moderation' array is empty - it doesn't render

Reply:
I don't think the dependency is needed? It looks like this is properly implemented with hooks to be an optional feature if both modules are enabled and does nothing if one of the modules isn't enabled :)

With that said we should be weary of renaming variables if their meaning doesn't really seem to change. It leads to larger code reviews and also makes understanding the code's history more difficult :)